src/shared/ui/survey/CheckBox/CheckBox.item.tsx

```diff
@@ -0,0 +1,110 @@
+import { FC, useMemo } from 'react';
+import { StyleSheet } from 'react-native';
+
+import { colors } from '@shared/lib';
+import { invertColor, handleReplaceBehaviourResponse } from '@shared/lib/utils';
+import {
+  Image,
+  Text,
+  XStack,
+  QuestionTooltipIcon,
+  Tooltip,
+  CheckBox,
+} from '@shared/ui';
+
+import { Item } from './types';
+
+type CheckBoxItemProps = {
+  colorPalette: boolean;
+  onChange: (itemValue: number) => void;
+  checked: boolean;
+} & Item;
+
+const CheckBoxItem: FC<CheckBoxItemProps> = ({
+  colorPalette,
+  onChange,
+  checked,
+  description,
+  image,
+  color,
+  value,
+  name,
+}) => {
+  const invertedColor =
+    colorPalette && color ? invertColor(color) : colors.primary;
+
+  const tooltipText = useMemo(
+    () => handleReplaceBehaviourResponse(description),
+    [description],
+  );
+  return (
+    <XStack
+      ai="center"
+      onPress={() => onChange(value)}
+      jc="space-between"
+      py="$3"
+      px="$5"
+      my="$1"
+      br={7}
+      bg={colorPalette ? color : 'none'}
+      minHeight="$7"
+      bbw={colorPalette ? 0 : 1}
+      bbc={colors.lighterGrey}
+    >
+      <XStack flex={1} ai="center">
+        {description && (
```

Comment:
I'd suggest to use !!description, otherwise we can get weird things in layout, js will return description value in case if it evaluated to false